### ISSUE TITLE
Add method for fast creation of communities only on status-mobile side

### ISSUE THREAD

**Issue by qfrank** [status-im/status-react#18715]
# Feature Issue
Add methods for fast creation of communities only on status-mobile side with combination of invoking exist backend `API`, e.g. [CreateCommunity](https://github.com/status-im/status-go/blob/9b10b29da26e8c4768dc62e3ddb5322b45ba8888/services/ext/api.go#L432) / [CreateCommunityChat](https://github.com/status-im/status-go/blob/9b10b29da26e8c4768dc62e3ddb5322b45ba8888/services/ext/api.go#L492) / [CreateCommunityCategory](https://github.com/status-im/status-go/blob/9b10b29da26e8c4768dc62e3ddb5322b45ba8888/services/ext/api.go#L653)

the code in [status-go implementation](https://github.com/status-im/status-go/blob/9b10b29da26e8c4768dc62e3ddb5322b45ba8888/protocol/messenger_community_for_mobile_testing.go#L22) will be removed once this feature is implemented on `status-mobile` side to apply these comments: [comment1](https://github.com/status-im/status-go/pull/3818#issuecomment-1923660012) / [comment2](https://github.com/status-im/status-go/pull/3818#issuecomment-1923690029).

relate [feature issue](https://github.com/status-im/status-mobile/issues/16362)
relate [status-go implementation](https://github.com/status-im/status-go/blob/9b10b29da26e8c4768dc62e3ddb5322b45ba8888/protocol/messenger_community_for_mobile_testing.go#L22) that can be referrenced. 
relate status-mobile commit id: `7b425c1a5301996afa28a079c159c80f952e05c3`

## User Story
As a QA automation engineer, I require the ability to utilize UI buttons within the e2e builds to instantaneously generate three distinct types of communities. This feature will significantly enhance the speed and efficiency of automation testing by utilizing frontend code to call upon status-go backend services.

## Description

1. Closed Community

  Name: closed community
  Description: A private community designed to verify membership requests.
  Logo: Any suitable image.
  Banner: Any suitable image.
  Colour: #887af9
  Community history service: Enabled
  Request to join: Required
  Any member can pin a message: Yes
  Channels:
      #general (no category)
      #cats (category: pets)
      #dogs (category: pets)
      #rules (category: household)
  Categories: pets, household

2. Open Community

  Name: open community
  Description: An accessible community allowing free join without requests.
  Logo: Any suitable image.
  Banner: Any suitable image.
  Colour: #26a69a
  Community history service: Enabled
  Request to join: Not required
  Any member can pin a message: No

3. Token-gated Community

  Name: SNT community
  Description: Entry requires holding 10 SNT on the Goerli network.
  Logo: Any suitable image.
  Banner: Any suitable image.
  Colour: #eab700
  Community history service: Enabled
  Request to join: Required
  Any member can pin a message: No
  PERMISSIONS: Holding 10 SNT on the Goerli network is a prerequisite for membership.

## Acceptance Criteria
The e2e and PR build environments must include three buttons, each dedicated to the creation of one of the specified community types using frontend code to invoke backend services, thereby streamlining the testing process for QA automation.

**Comment by qfrank** [comment]
cc @cammellos

**Comment by ilmotta** [comment]
@qfrank is this issue still relevant? If not, could you close it please?

**Comment by qfrank** [comment]
> [@qfrank](https://github.com/qfrank) is this issue still relevant? If not, could you close it please?

I think @osmaczko would be happy if we implement it. relate [comment](https://github.com/status-im/status-go/pull/3818#issuecomment-1923660012) 
Someone said he will implement it, but something unknown happened you know ): @ilmotta

**Comment by ilmotta** [comment]
I agree with @osmaczko as well in his comment, that we should use the public API to create test data for testing purposes.

Thanks @qfrank, I just added the `tech-debt` label and the issue can remain open no worries.
